app/components/agenda/agendaitem/agendaitem-case-panel/agendaitem-case-panel-edit.js

```diff
@@ -79,5 +86,6 @@ export default class AgendaitemCasePanelEdit extends Component {
       }
     }
     this.args.onSave();
+    this.router.refresh();
```

Comment:
router refresh to update 1 icon in the side bar is very load heavy.
Let me see if I can fix it to update better

Reply:
Managed to fix this quite easily without a full page refresh